addons/hr_expense/static/tests/views/expense_line_widget_tests.js

```diff
@@ -56,7 +57,14 @@ QUnit.module("Views", {}, function () {
                 step(`${route} - ${JSON.stringify(args)}`);
             },
         });
-        await assertSteps(['/mail/action - {"init_messaging":true,"failures":true}']);
+        await assertSteps([
+            `/mail/action - ${JSON.stringify({
+                init_messaging: true,
+                failures: true,
+                systray_get_activities: true,
```

Comment:
Note: extra param appearing in tests throughout the diff is due to removal of activity from `ROUTES_TO_IGNORE`, not because extra fetch.

This is actually a reduction as without this PR, there would be an extra entry in the array.